pkgs/tools/networking/openssh/default.nix

```diff
@@ -87,6 +87,8 @@ stdenv.mkDerivation rec {
     ++ optional stdenv.isDarwin "--disable-libutil"
     ++ optional (!linkOpenssl) "--without-openssl";
 
+  buildFlags = [ "SSH_KEYSIGN=/run/wrappers/bin/ssh-keysign" ];
```

Comment:
Can we just do SSH_KEYSIGN=ssh-keysign? Non-NixOS users will not have this directory

Reply:
If not, at least make the path an argument to the derivation?

Reply:
> Can we just do SSH_KEYSIGN=ssh-keysign? Non-NixOS users will not have this directory

Nope, not without patching [the code that uses that information](https://github.com/openssh/openssh-portable/blob/8de52eb224143783a49f9bddd9ab7800022a8276/sshconnect2.c#L1883). If we're picking that solution, I'm perfectly fine with it and I'll amend my patch to do that instead.

> If not, at least make the path an argument to the derivation?

I considered this, but wasn't sure what to name the parameter. Perhaps `sshKeySignPath`?

Reply:
> > Can we just do SSH_KEYSIGN=ssh-keysign? Non-NixOS users will not have this directory
> 
> Nope, not without patching [the code that uses that information](https://github.com/openssh/openssh-portable/blob/8de52eb224143783a49f9bddd9ab7800022a8276/sshconnect2.c#L1883). If we're picking that solution, I'm perfectly fine with it and I'll amend my patch to do that instead.

Are there security implications of either approach? Nothing bad would happen if a malicious ssh-keysign was in PATH, right?

> > If not, at least make the path an argument to the derivation?
> 
> I considered this, but wasn't sure what to name the parameter. Perhaps `sshKeySignPath`?

Nit: should be `sshKeysignPath` to match the build flag. Also should make sure to expose it as a passthru.

Reply:
> Are there security implications to either approach? Nothing bad would happen if a malicious ssh-keysign was in PATH, right?

I don't *think* so. As far as I understand, it would be roughly analogous to having a rogue `SSH_AUTH_SOCK`. But we would certainly be deviating from what OpenSSH does on any other platform, and I'm somewhat reluctant to do so.

> Nit: should be sshKeysignPath to match the build flag. Also should make sure to expose it as a passthru.

ACK! Patch amendment coming up.